src/server/db/migration/migrations/sqls/20221003094432-table-activity-log-up.sql

```diff
@@ -0,0 +1,14 @@
+drop table if exists activity_log;
+
+create table activity_log
+(
+    "time"           timestamp default timezone('UTC'::text, now()) not null,
+    message          text,
```

Comment:
`message` not null?

Reply:
you're right. it was nullable in the old fra_audit but it should be not null here.

<img width="663" alt="Schermata 2022-10-04 alle 19 03 51" src="https://user-images.githubusercontent.com/10047945/193881975-f4edc214-db7d-4092-bc55-0b161855b7e3.png">